src/arrays/programs_involving_data_sequences/hard/q8/MultItUp.java

```diff
@@ -0,0 +1,38 @@
+package arrays.programs_involving_data_sequences.hard.q8;
+
+/*
+Write a Java program that asks the user to enter 10 integers, stores them in an array, and passes that array to a method
```

Comment:
Number of inputs needs to be higher, also could you check if moving the 
`for (int i = 0; i < 10; i++)
            arr[i] = in.nextInt();
        productMaker(arr);`
after the "The result of multiplied array is:" statement to check if the test still approves it. If it does, something about it needs to change since they can just for loop catch and print the values without storing them.

Reply:
They won't be able to because we are using the method that takes an array.